### DISCUSSION TITLE
Feature request: Perigee/Apogee & node marks in Moon orbit.

### DISCUSSION THREAD

**Discussion by MartinLewicki** [Stellarium/stellarium#1917] [Ideas]
Teaching aid to for demonstrating lunar orbit features.

Martin

**Comment by gzotti** [comment]
This is at least problematic because we don't visualize any mean Lunar orbit in terms of nodes and apsides, just a series of positions of the Moon in its orbit-like track. (This is true also for the major planets.)
AstroCalc allows to visualize the distances from which you may see the respective times. (Graphs tab. See distance and e.g. phase if you are one of those who gives anything to "supermoons"... )

**Comment by MartinLewicki** [comment]
Hi  Georg

Yes, I see the point. Given the lunar orbit is constantly perturbed, the
nodes and apsides wander about their mean points. However, these algorithms
exist in Meeus Astronomical Algorithms. Perhaps you may consider this in
the future.

Thank you for a most useful full program. it is a valuable resource for
teaching at our planetarium.

Martin


Martin Lewicki
Adelaide Planetarium | Astronomy Educator
UniSA STEM | University of South Australia
Mawson Lakes
GPO Box 2471, Adelaide
South Australia 5001
Ph: +618 830*2 3027*
email ***@***.***
web: www.unisa.edu.au/planetarium/
